ptbr/data.py

```
@@ -0,0 +1,224 @@
+
+    dataset = load_dataset(file_or_repo, split=split)
+    available_columns = getattr(dataset, "column_names", None)
+    if available_columns is not None:
+        missing_columns = [col for col in (text_column, ner_column) if col not in available_columns]
+        if missing_columns:
+            missing = ", ".join(repr(col) for col in missing_columns)
+            available = ", ".join(repr(col) for col in available_columns)
+            raise ValueError(
+                f"Missing required column(s): {missing}. "
+                f"Available columns for split '{split}': {available}"
+            )
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)